bindings/rust/s2n-tls/src/error.rs

```diff
@@ -136,6 +147,12 @@ impl<T: Fallible> Pollable for T {
 }
 
 impl Error {
+    // Previously we referenced InvalidInput via Error::InvalidInput.
+    // Keep this naming.
+    // TODO: Update this + all references to all upper case.
+    #[allow(non_upper_case_globals)]
+    pub const InvalidInput: Error = Self(Context::InvalidInput);
```

Comment:
since we are pre 1.0 why not just change this?

Reply:
I plan to, but in a separate PR. It's a lot of find and replace that would confuse the diff.